extras/secure-email.rst

```diff
@@ -1,59 +1,78 @@
-   Administrators can learn more
-   `here <https://admin-docs.zammad.org/en/latest/system/integrations/smime.html>`_.
+   Administrators can learn more here:
+      * :admin-docs:`PGP </system/integrations/pgp/index.html>`
+      * :admin-docs:`S/MIME </system/integrations/smime/index.html>`
 
-What is S/MIME?
----------------
+Overview
+--------
 
-S/MIME is the most widely-supported method for secure email communication.
-With S/MIME, you can exchange **signed** and **encrypted** messages with others.
+PGP and S/MIME are the most widely-supported methods for secure email
+communication. With each of the systems, you can exchange **signed** and 
+**encrypted** messages with others.
 
 Signing
-   is proof that a message hasn’t been tampered with or sent by an impersonator.
+   is a proof that a message hasn’t been manipulated on its way.
 
-   In other words, it guarantees a message’s **integrity** and **authenticity**.
+   In other words, it guarantees message **integrity** and **authenticity**.
 
 Encryption
-   scrambles a message so that it can only be unscrambled by the intended recipient.
+   scrambles a message so that it can only be unscrambled by the intended
+   recipient.
 
-   In other words, it guarantees **privacy** and **data security**.
+   In other words, it guarantees message **privacy** and **data security**.
 
-Overview
---------
+.. note:: 🤝 **PGP and S/MIME are only working if the other party is using it,
+   too.**
 
-.. note:: 🤝 **S/MIME only works if the other party is using it, too.**
+   Your administrator is responsible for adding all the necessary certificates
+   in Zammad’s admin panel.
 
-   Your administrator is responsible for
-   adding all the necessary certificates in Zammad’s admin panel.
+.. note:: In special cases it is possible that both systems are configured in
+   your system *and* a customer is using both, as well. In this case, you have
+   an additional button to switch between PGP and S/MIME. Otherwise, you just
+   see the 🔒 **Encrypt** and ✅ **Sign** buttons.
+
+.. figure:: /images/extras/secure-email/pgp_and_smime.png
+   :alt: Screenshot of ticket creation with configured PGP and S/MIME 
+   :scale: 50%
+   :align: center
+
+   Ticket creation with configured PGP *and* S/MIME and available
+   certificates/keys.
 
 📬 Incoming
 ^^^^^^^^^^^
 
-The 🔒 and ✅ icons at the top of a message indicate its S/MIME status.
+The 🔒 and ✅ icons at the top of a message indicate its encryption and signing
+status.
 
 .. figure:: /images/extras/secure-email/checking-security-metadata.gif
-   :alt: Screencast showing on how to verify used certificates
+   :alt: Screencast showing details of encryption and signing status
    :scale: 50%
    :align: center
 
    Click on an incoming message to expand its details.
-   Hover over the security status to show a certificate/CA summary.
+   Hover over the security status to show details.
 
 .. list-table:: Status Icons (Incoming)
    :widths: 5 45
```

Comment:
We need to adjust one icon here and accompanying texts as some changes were also applied to this part:

* `open-lock` => This icon should be exchanged with an icon of a closed lock with an X on it (attached). The meaning of this state is that the decryption of the encrypted message has failed (more info will be provided on hover).
* `not-signed` => The meaning of this state is that the signature verification of the signed message has failed (more info will be provided on hover).

Again word `contents` should be exchanged with `content` everywhere, since this is an uncountable noun. (Contents is used primarily for table of contents, for example.)